config.py

```diff
@@ -89,7 +89,7 @@ class DevelopmentConfig(Config):
 
     ACCOUNT_SERVICE_URL = os.getenv("ACCOUNT_SERVICE_URL", "http://localhost:8082/surveys/todo")
     ACCOUNT_SERVICE_LOG_OUT_URL = os.getenv("ACCOUNT_SERVICE_LOG_OUT_URL", "http://localhost:8082/sign-in/logout")
-    EQ_URL = os.getenv("EQ_URL", "http://localhost:8086/session?token=")
+    EQ_URL = os.getenv("EQ_URL", "http://localhost:8086/complete-eq?token=")
```

Comment:
why this change?  that's not the right url

Reply:
ohh.. needs to revert this.. this was not suppose to go in